transformers/tls.py

```diff
@@ -266,19 +268,19 @@ def analyze_tls(item: dict, collection_data: datetime.datetime) -> dict:
         "tls_expired_chain": expired_chain,                         # Chain already expired at time of collection
         "tls_total_extension_count": total_extension_count,         # Total number of extensions in certificate
         "tls_critical_extensions": critical_extensions,             # Total number of critical extensions in certificate
-        "tls_have_policies": have_policies,                         # Number of certificates enforcing specific encryption policy
-        "tls_percentage_of_policies": percentage_of_policies,       # Percentage of certificates enforcing specific encryption policy
-        "tls_unknown_policy_crt_count": unknown_usage,              # How many cerificates uses unknown (not X509v3, not version 1, not version 2) policy
-        "tls_anypolicy_crt_count": X_509_used_cnt,                  # Number of certificates enforcing X509v3 policy
-        "tls_joint-iso-itu_policy_crt_count": version_2_used_cnt,   # Number of certificates enforcing version 2 policy
-        "tls_iso_policy_crt_count": version_1_used_cnt,             # Number of certificates enforcing version 1 policy
-        "tls_subject_crt_count": subject_count,                     # How many subjects can be found in SAN extension ( can be linked to phishing)       
-        "tls_server_auth_certs": server_auth,                       # How many certificates are used for server authentication (can be simultanously used for client authentication)      
-        "tls_client_auth_certs": client_auth,                       # How many certificates are used for client authentication
+        "tls_with_policies_crt_count": any_policy_cnt,              # Number of certificates enforcing specific encryption policy
+        "tls_percentage_crt_with_policies": percentage_of_policies, # Percentage of certificates enforcing specific encryption policy
+        "tls_iso_v1_policy_crt_count": version_1_used_cnt,          # Number of certificates enforcing version 1 policy
+        "tls_iso_v2_policy_crt_count": version_2_used_cnt,          # Number of certificates enforcing version 2 policy
+        "tls_unknown_policy_crt_count": unknown_policy_cnt,         # How many cerificates uses unknown (not X509v3, not version 1, not version 2) policy
+        "tls_x509-policy_crt_count": X_509_used_cnt,                # Number of certificates enforcing X509v3 policy
```

Comment:
https://datatracker.ietf.org/doc/html/rfc5280#section-4.2.1.4
https://oidref.com/2.5.29.32.0
We should adjust the regex instead of renaming things just to match it.